### PR TITLE
upd tvl

### PR DESCRIPTION
**Pull request by BadConfig**
We add new contract to tvl measurement as we've released new ETF

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/arcanum: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:217:11

**Comment by llamatester** [comment]
The adapter at projects/arcanum exports TVL: 
        
 
 
    arbitrum                  331.3372607540959
    
    total                    331.3372607540959

**Comment by BadConfig** [comment]
> Error while running adapter at projects/arcanum:
> 
> ```
> Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
> Truncated error stack:
> 
>     at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:217:11
> ```

fixed

File: projects/arcanum/index.js [hunk 1]
```diff
@@ -10,11 +10,18 @@ const ASSETS_CONTRACTS = [
...
+];
+
 
 module.exports = {
-    methodology: 'counts the quantities of all tokens in multipool contracts.',
+    methodology: 'counts the quantities of all tokens in all multipool contracts.',
     start: 1000235,
     arbitrum: {
-        tvl: sumTokensExport({ owner: ARBI_CONTRACT, tokens: ASSETS_CONTRACTS }),
+        tvl: sumTokensExport({ owners: [ARBI_CONTRACT, SPI_CONTRACT], tokens: ARBI_ASSETS_CONTRACTS.concat(SPI_ASSETS_CONTRACTS) })
```
Comment (g1nt0ki) [inline projects/arcanum/index.js:25]: can you change the line to below? it is bit more efficient

    tvl: sumTokensExport({ ownerTokens: [[ARBI_ASSETS_CONTRACTS, ARBI_CONTRACT], [SPI_ASSETS_CONTRACTS, SPI_CONTRACT]], })

**Review by g1nt0ki — COMMENTED** [review]
.

**Comment by llamatester** [comment]
The adapter at projects/arcanum exports TVL: 
        
 
 
    arbitrum                  331.44306952904566
    
    total                    331.44306952904566

File: projects/arcanum/index.js (hunk 1 above)
Comment (BadConfig) [reply projects/arcanum/index.js]: > can you change the line to below? it is bit more efficient
> 
> ```
> tvl: sumTokensExport({ ownerTokens: [[ARBI_ASSETS_CONTRACTS, ARBI_CONTRACT], [SPI_ASSETS_CONTRACTS, SPI_CONTRACT]], })
> ```

done

**Pull request merged**